content/batch/workflow-advanced.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Advanced Batch Workflow"
+date: 2018-11-18T00:00:00-05:00
+weight: 70
+draft: false
+---
+
+### Advanced Batch Workflow
+
+Let's take a look at a more complex workflow, involving passing artifacts between jobs, multiple dependencies, etc.
+
```

Comment:
Save the below manifest as 'teardrop.yaml' using your favorite editor.

Reply:
Updated in future commit.